lib/Dialect/TTNN/Utils/Utils.cpp

```diff
@@ -59,8 +57,6 @@ mlir::tt::TensorMemoryLayout toTTTensorMemoryLayout(
     return ::mlir::tt::TensorMemoryLayout::BlockSharded;
   case ttnn::TensorMemoryLayout::SingleBank:
     return ::mlir::tt::TensorMemoryLayout::SingleBank;
-  case ttnn::TensorMemoryLayout::None:
```

Comment:
Add `llvm_unreachable`

Reply:
The idea is that it provides a clear error message in case someone adds an enum at some point, similar to the `toTTMemorySpace` fn below.

Reply:
Compiler will generate error regardless since that new enum won't be handled in switch.